restaurant/models.py

```diff
@@ -9,11 +9,9 @@
 
 GEOCODERS = [Nominatim, GoogleV3, ArcGIS, OpenMapQuest, GeocoderDotUS]
 
-VEGO_RESTO_URL = "https://vegoresto.fr/%s"
-
 class Restaurant(models.Model):
     vegoresto_id = models.BigIntegerField(unique=True)
-    vegoresto_url = models.TextField(null=True)
+    vegoresto_url = models.URLField(null=True, blank=True)
```

Comment:
I ran the "make-migration" and "migrate" commands (which worked).
(Although I didn't check the sqlite struct diff it may have implied).

But I had several *other* unversioned migration files (from
other runs of `create-migration` maybe?) than the one related to the
URLField change.

All (3 or 4 in total) passed correctly in my case, but can't say for
sure it will in somewhat different setups